xz-pgo/PKGBUILD

```diff
@@ -0,0 +1,59 @@
+# Maintainer: Pierre Schmitz <pierre@archlinux.de>
+# Contributor: François Charette <firmicus@gmx.net>
+# PGO Version: Laio Seman <laio [at] ieee.org>
+
+pkgname=xz
+pkgver=5.4.5
+pkgrel=101
```

Comment:
typo?

Reply:
Personal pkgrel for my custom packages, for laziness of doing custom repo. :D